packages/entity-list/src/modules/searchForm/sagas.js

```diff
@@ -388,3 +399,85 @@ export function* resetSearchFields() {
   yield call(loadSearchForm, true)
   yield call(resetSearch)
 }
+
+export function* loadSearchAsQuery() {
+  const {where: condition, filter: filters} = yield call(getBasicQuery, false, false)
```

Comment:
Idea:
Can we use here `getSearchViewQuery` directly instead of `getBasicQuery(false, false)` and therefore remove the second optional parameter `regardQueryView` in `getBasicQuery`?